README.md

```diff
@@ -4,39 +4,62 @@
 [![Build Status](https://circleci.com/gh/OpenZeppelin/openzeppelin-contracts.svg?style=shield)](https://circleci.com/gh/OpenZeppelin/openzeppelin-contracts)
 [![Coverage Status](https://codecov.io/gh/OpenZeppelin/openzeppelin-contracts/graph/badge.svg)](https://codecov.io/gh/OpenZeppelin/openzeppelin-contracts)
 
-**OpenZeppelin Contracts is a library for secure smart contract development.** It provides implementations of standards like ERC20 and ERC721 which you can deploy as-is or extend to suit your needs, as well as Solidity components to build custom contracts and more complex decentralized systems.
+*A library for secure smart contract development.* Build on a solid foundation of community-vetted code.
 
-## Install
+ * Implementations of standards like [ERC20](https://docs.openzeppelin.com/contracts/erc20) and [ERC721](https://docs.openzeppelin.com/contracts/erc721).
+ * Flexible [role-based permissioning](https://docs.openzeppelin.com/contracts/access-control) scheme.
+ * Reusable [Solidity components](https://docs.openzeppelin.com/contracts/utilities) to build custom contracts and complex decentralized systems.
+ * First-class integration with the [Gas Station Network](https://docs.openzeppelin.com/contracts/gsn) for systems with no gas fees!
+ * Audited by leading security firms.
 
-```
-npm install @openzeppelin/contracts
+## Overview
+
+### Installation
+
+```console
+$ npm install @openzeppelin/contracts
 ```
 
-OpenZeppelin Contracts features a stable API, which means your contracts won't break unexpectedly when upgrading to a newer minor version. You can read ṫhe details in our [API Stability] document.
+OpenZeppelin Contracts features a [stable API](https://docs.openzeppelin.com/contracts/releases-stability#api-stability), which means your contracts won't break unexpectedly when upgrading to a newer minor version.
 
-## Usage
+### Usage
 
-To write your custom contracts, import ours and extend them through inheritance.
+Once installed, you can use the contracts in the library by importing them:
 
 ```solidity
 pragma solidity ^0.5.0;
 
-import "@openzeppelin/contracts/token/ERC721/ERC721Full.sol";
-import "@openzeppelin/contracts/token/ERC721/ERC721Mintable.sol";
+import "@openzeppelin/contracts/ownership/Ownable.sol";
 
-contract MyNFT is ERC721Full, ERC721Mintable {
-  constructor() ERC721Full("MyNFT", "MNFT") public {
-  }
+contract MyContract is Ownable {
```

Comment:
I was more fond of the ERC721 example. Too many people know about `Ownable` already. Don't you think it illustrated the use of the library much better? (Issues in Solidity 0.6 notwithstanding...)

Reply:
I'd say the opposite, the old example was weird in that it did nothing other than patch `ERC721Full`'s lack of a constructor :stuck_out_tongue: 

I'm not against changing this for something more interesting though.

Reply:
> did nothing other than patch `ERC721Full`'s lack of a constructor

That's not true. It defines the name and symbol, _and_ mixes in the mintable flavor.

Reply:
Heh, true. What I failed to convey is I'm not a fan of the old example not showing that Contracts can be used as building blocks for the user's code, as opposed to ready-made solutions can are just mixed in.

Reply:
It's an interesting discussion. Lately I feel that the latter is our more valuable offering.

Reply:
I'm bringing back the old example to get this merged.